diff --git a/media/base/media_constants.cc b/media/base/media_constants.cc
index 966ac39eba..94bc7e8749 100644
--- a/media/base/media_constants.cc
+++ b/media/base/media_constants.cc
@@ -138,10 +138,10 @@ const char kAv1FmtpTier[] = "tier";
 const int kDefaultVideoMaxFramerate = 60;
 // Max encode quantizer for VP8/9 and AV1 encoders assuming libvpx/libaom API
 // range [0, 63]
-const int kDefaultVideoMaxQpVpx = 56;
-const int kDefaultVideoMaxQpAv1 = 52;
+const int kDefaultVideoMaxQpVpx = 20;
+const int kDefaultVideoMaxQpAv1 = 20;
 // Max encode quantizer for H264/5 assuming the bitstream range [0, 51].
-const int kDefaultVideoMaxQpH26x = 51;
+const int kDefaultVideoMaxQpH26x = 20;
 
 const size_t kConferenceMaxNumSpatialLayers = 3;
 const size_t kConferenceMaxNumTemporalLayers = 3;
diff --git a/modules/video_coding/codecs/av1/libaom_av1_encoder.cc b/modules/video_coding/codecs/av1/libaom_av1_encoder.cc
index 8c64a56eed..5e8ea2b1d9 100644
--- a/modules/video_coding/codecs/av1/libaom_av1_encoder.cc
+++ b/modules/video_coding/codecs/av1/libaom_av1_encoder.cc
@@ -65,8 +65,8 @@ namespace webrtc {
 namespace {
 
 // Encoder configuration parameters
-constexpr int kMinQp = 10;
-constexpr int kMinQindex = 40;  // Min qindex corresponding to kMinQp.
+constexpr int kMinQp = 0;
+constexpr int kMinQindex = 0;   // Min qindex corresponding to kMinQp.
 constexpr int kUsageProfile = AOM_USAGE_REALTIME;
 constexpr int kLowQindex = 145;   // Low qindex threshold for QP scaling.
 constexpr int kHighQindex = 205;  // High qindex threshold for QP scaling.
diff --git a/modules/video_coding/codecs/vp8/libvpx_vp8_encoder.cc b/modules/video_coding/codecs/vp8/libvpx_vp8_encoder.cc
index 5446d837ec..8c3c0c86ce 100644
--- a/modules/video_coding/codecs/vp8/libvpx_vp8_encoder.cc
+++ b/modules/video_coding/codecs/vp8/libvpx_vp8_encoder.cc
@@ -628,8 +628,7 @@ int LibvpxVp8Encoder::InitEncode(const VideoCodec* inst,
   vpx_configs_[0].g_pass = VPX_RC_ONE_PASS;
   // Handle resizing outside of libvpx.
   vpx_configs_[0].rc_resize_allowed = 0;
-  vpx_configs_[0].rc_min_quantizer =
-      codec_.mode == VideoCodecMode::kScreensharing ? 12 : 2;
+  vpx_configs_[0].rc_min_quantizer = 0;
   if (inst->qpMax >= vpx_configs_[0].rc_min_quantizer) {
     qp_max_ = inst->qpMax;
   }
diff --git a/modules/video_coding/codecs/vp9/libvpx_vp9_encoder.cc b/modules/video_coding/codecs/vp9/libvpx_vp9_encoder.cc
index d941cd6f00..a2b05c2a92 100644
--- a/modules/video_coding/codecs/vp9/libvpx_vp9_encoder.cc
+++ b/modules/video_coding/codecs/vp9/libvpx_vp9_encoder.cc
@@ -674,9 +674,8 @@ int LibvpxVp9Encoder::InitEncode(const VideoCodec* inst,
   config_->rc_dropframe_thresh = codec_.GetFrameDropEnabled() ? 30 : 0;
   config_->rc_end_usage = VPX_CBR;
   config_->g_pass = VPX_RC_ONE_PASS;
-  config_->rc_min_quantizer =
-      codec_.mode == VideoCodecMode::kScreensharing ? 8 : 2;
-  config_->rc_max_quantizer = 52;
+  config_->rc_min_quantizer = 0;
+  config_->rc_max_quantizer = 20;
   config_->rc_undershoot_pct = 50;
   config_->rc_overshoot_pct = 50;
   config_->rc_buf_initial_sz = 500;
